src/briefcase/platforms/windows/__init__.py

```diff
@@ -354,10 +374,23 @@ def package_app(
         except subprocess.CalledProcessError as e:
             raise BriefcaseCommandError(f"Unable to link app {app.app_name}.") from e
 
-        if sign_app:
-            self.logger.info("Signing MSI...", prefix=app.app_name)
-            self.sign_file(
-                app=app,
-                filepath=self.distribution_path(app),
-                **sign_options,
-            )
+    def _package_zip(self, app):
+        """Package the app as simple zip file."""
+
+        self.logger.info("Building zip file...", prefix=app.app_name)
+        with self.input.wait_bar("Packing..."):
+            source = self.bundle_path(app) / self.packaging_root  # /src
+            try:
+                os.mkdir(self.dist_path)  # /dist
+            except FileExistsError:
+                pass
+
+            with ZipFile(self.distribution_path(app), "w", ZIP_DEFLATED) as archive:
+                for path, _, files in os.walk(source):
```

Comment:
We generally prefer the `pathlib` form of these commands to the `os` form - so `source.glob("*/**")` in this case. Similar updates are needed for the other `os` calls in this block.

Reply:
OK, did so.